examples/README.md

```diff
@@ -0,0 +1,20 @@
+## Demo Notebook
+
+We can launch a jupyter notebook to try out the library or transforms or to build a usecase.
```

Comment:
We need to either a) tell the user where to get an input data set and where to put it or b) include one in the repo.  However, if (b), it should not be large - say not larger than a couple of MB if possible.

Reply:
Addressed in the rebased pull req on another branch https://github.com/IBM/data-prep-lab/pull/89

Updated Notebook with instructions to get data